i18n/en.json

```diff
@@ -1038,11 +1038,11 @@
   },
   {
     "id": "api.file.file_exists.exists_local.app_error",
-    "translation": "Unable to know if the file exists. An error ocurred when trying to check file existency."
+    "translation": "Unable to know if the file exists. An error ocurred when trying to check file existence."
```

Comment:
Can we simplify this to just, `An error occurred when trying to check if the file exists.` The term `file existence` is better than `file existency`, for sure, but both are still complicated English.

Reply:
Please take a look at the discussion I started about this topic. Maybe this would be wort ha second step.
https://pre-release.mattermost.com/core/pl/b5gpxp5rqtr8deox1pube88b1w

Reply:
With the harmonization plan, perhaps this becomes the simpler, `Unable to check if the file exists.`